README.md

```
@@ -29,4 +29,36 @@ Using generated security password: 38915130-c6d6-46d3-93f7-1f93ad77dc1d
\ No newline at end of file
+Using generated security password: 38915130-c6d6-46d3-93f7-1f93ad77dc1d
+
+
+
+
+
+
+
+
+
+## Environment Setup
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: atx_closed; Actual: atx

(MD003, heading-style)